packages/dashboard/src/components/storyMenu/stories/index.js

```diff
@@ -51,8 +50,7 @@ export const _default = () => {
       <StoryMenu
         onMoreButtonSelected={setContextMenuId}
         contextMenuId={contextMenuId}
-        onMenuItemSelected={(item, story) => {
-          actions('onClick ', item.label, story.id);
+        onMenuItemSelected={() => {
```

Comment:
I just removed this action as a whole because I don't think we can trigger `onMenuItemSelected` from this story.